meta-luneos/recipes-support/libgbinder/libgbinder-radio.bb

```diff
@@ -3,7 +3,7 @@
 DESCRIPTION = "Library used to interact with Android's binder module."
 LICENSE = "BSD-3-Clause"
 SECTION = "webos/support"
-LIC_FILES_CHKSUM = "file://src/radio_instance.c;beginline=1;endline=31;md5=ed07559b4ce676bd0c42c9416f003aff"
+LIC_FILES_CHKSUM = "file://src/radio_instance.c;beginline=1;endline=31;md5=d2f669b71d1c54b4a37962a08248b0a7"
```

Comment:
Suggest to make it: LIC_FILES_CHKSUM = "file://LICENSE;md5=022248aa7588481d6a6c365c101d6249"